signature/signature_test.go

```diff
@@ -5,6 +5,7 @@ import (
 	"io/ioutil"
 	"testing"
 
+	_ "github.com/containers/image/openshift"
```

Comment:
wrong, file this is needed in `policy_config_test.go` where we use the `atomic` transport in https://github.com/containers/image/blob/master/signature/policy_config_test.go#L242. `docker` and `dir` are already imported in that test file so we just need this or it fails with:
```
--- FAIL: TestPolicyUnmarshalJSON (0.00s)
        Error Trace:    policy_config_test.go:255
        Error:          Received unexpected error "Unknown key \"atomic\""
```
Btw, fixed